h/static/scripts/admin-users.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function AdminUsersController(element, window_) {
+  this._form = element.querySelector('#js-users-delete-form');
+
+  function confirmFormSubmit() {
+    return window_.confirm("This will permanently delete all the user's data. Are you sure?");
+  }
+
+  if(this._form) {
```

Comment:
Nit, should have a space between the `if` and `(`. You can use `jscs <filename>` to check the coding style for a file and `jscs --fix <filename>` to fix up trivial issues automatically.